content/docs/install/10. podman.md

```diff
@@ -64,7 +58,8 @@ Still confused about Docker? Check out [this FAQ](/faq/server#im-still-confused-
 >  -v </path/to/config>:/config \
 >  -v </path/to/metadata>:/metadata \
 >  -v </path/to/audiobooks>:/audiobooks \
->  -v </path/to/podcasts>:/podcasts \
```

Comment:
ABS supports podcast libraries. We do not want to remove this from the example commands for setting up a server

Reply:
I have added the podcast volume mount in the podman guide.